### DISCUSSION TITLE
Scroll need to collaborate with Tier-1 CEX to maintain deposits/withdrawals.

### DISCUSSION THREAD

**Discussion by pr1museth** [scroll-tech/scroll#1159] [Ideas]
**In order to expand and attract new communities Scroll team should add the deposits/withdrawals from top tier exchanges.**

It'll help users to avoid bridging from ETH, will cut some costs and new users can easily join and use Scroll ecosystem.
Nowadays when we approach to global expansion and mass adoption of crypto it should be simple to start using scroll for everyone.

![Scroll](https://github.com/scroll-tech/scroll/assets/121545087/8d347f43-f194-4db8-90e7-3d0569df1bfe)

Such CEXs like Binance/OKX/Bybit and others are big part of the cryptomarket and sometimes it's more easier to transfer from exchange to wallet without paying more or risks with phishing bridges. 
I think it's musthave for building Scroll ecosystem.

![C](https://github.com/scroll-tech/scroll/assets/121545087/b2ca00a8-3c55-4244-a17c-a5283ff8cca6)

**Comment by farfadet82** [comment]
Agreed but it's probably too early - I'm sure it's in the pipe though. Look at Arbitrum and co, no need to bridge as you can transfer from CEX

**Reply by kalistratina** [reply]
It's good time. Before mainet people can withdraw eth from cex
